spec/services/notifications_list_spec.rb

```diff
@@ -35,6 +35,26 @@
 
         expect(NotificationsList.for(application_choice, include_ratifying_provider: true, event: :offer_accepted).to_a).to match_array([training_provider_user, ratifying_provider_user])
       end
+
+      it 'returns a provider user who is a member of the training and ratifying providers without duplicates' do
+        ratifying_provider = create(:provider)
+        training_provider = create(:provider)
+        provider_user = create(:provider_user, send_notifications: true, providers: [training_provider, ratifying_provider])
```

Comment:
```suggestion
        provider_user = create(:provider_user, providers: [training_provider, ratifying_provider])
```

If we have new prefs we don't need `send_notifications: true`

Reply:
I think its good to keep them aligned until we remove the implementation soon (hopefully next sprint) to avoid confusion